Library/Homebrew/github_runner_matrix.rb

```diff
@@ -181,7 +181,7 @@ def generate_runners!
       )
       @runners << create_runner(:macos, :arm64, spec, macos_version)
 
-      next if macos_version > NEWEST_HOMEBREW_CORE_INTEL_MACOS_RUNNER
+      next if !@all_supported && macos_version > NEWEST_HOMEBREW_CORE_INTEL_MACOS_RUNNER
```

Comment:
Could be worth renaming `NEWEST_HOMEBREW_CORE_INTEL_MACOS_RUNNER`, though not really sure what name suits.

Reply:
yeah though not really sure either

Reply:
`NEWEST_HOMEBREW_CORE_INTEL_MACOS_RUNNER_MOST_OF_THE_TIME`